### PR TITLE
ASimDnsMicrosoftSysmon.yaml-22

### PR DESCRIPTION
**Pull request by Alekhya0824**
Required items, please complete
   
   Change(s):
   - Created a new file for WindowsEvent and added WindowsEvent parser in it. In Event file removed WindowsEvent parser and in ASimDns added WindowsEvent Parser name  and same changes have done for vim as well.

   Reason for Change(s):
   - To Split the WindowsEvent and Event parser separately

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml [hunk 1]
```diff
@@ -121,44 +121,6 @@ ParserQuery: |
       | where  (responsecodename=="*" or DnsResponseCodeName has responsecodename) // -- filter is not optimized
       // --
     };
-  let ParsedDnsEvent_WindowsEvent =(
```
Comment (vakohl) [inline Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml:124]: please keep these filters in separate lines:
 starttime:datetime=datetime(null), endtime:datetime=datetime(null)
    , srcipaddr:string='*'
    , domain_has_any:dynamic=dynamic([]) 
    , responsecodename:string='*', response_has_ipv4:string='*'
    , response_has_any_prefix:dynamic=dynamic([]) , eventtype:string='Query'
    , disabled:bool=false

Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml:124]: Done

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  , 9022,'BADTRUNC'
+  , 9023,'BADCOOKIE'
+  , 1460, 'TIMEOUT'
+  ];
+  let ParsedDnsEvent_WindowsEvent =(
+    starttime:datetime=datetime(null), endtime:datetime=datetime(null)
+    , srcipaddr:string='*'
+    , domain_has_any:dynamic=dynamic([]) 
+    , responsecodename:string='*', response_has_ipv4:string='*'
+    , response_has_any_prefix:dynamic=dynamic([]) , eventtype:string='lookup'
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml:79]: Done

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml [hunk 3]
```diff
@@ -0,0 +1,165 @@
+Parser:
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml:1]: Done

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml:1]: Done

File: Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmonWindowsEvent.yaml [hunk 5]
```diff
@@ -0,0 +1,108 @@
+Parser:
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmonWindowsEvent.yaml:1]: Done

File: Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmon.yaml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmon.yaml:1]: Done

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+  Title: DNS activity ASIM filtering parser for Sysmon for Windows
+  Version: '0.5.1'
+  LastUpdated: June 7 2024
+Product:
+  Name: Microsoft Windows Events Sysmon
+Normalization:
+  Schema: Dns
+  Version: '0.1.6'
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml:9]: Done

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml [hunk 8]
```diff
@@ -0,0 +1,165 @@
+Parser:
+  Title: DNS activity ASIM filtering parser for Sysmon for Windows
+  Version: '0.5.1'
+  LastUpdated: June 7 2024
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmonWindowsEvent.yaml:4]: Done

File: Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmonWindowsEvent.yaml [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+  Title: DNS activity ASIM parser for Sysmon for Windows
+  Version: '0.5.1'
+  LastUpdated: June 7 2024
+Product:
+  Name: Microsoft Windows Events Sysmon
+Normalization:
+  Schema: Dns
+  Version: '0.1.6'
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmonWindowsEvent.yaml:9]: Done

File: Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmonWindowsEvent.yaml [hunk 10]
```diff
@@ -0,0 +1,108 @@
+Parser:
+  Title: DNS activity ASIM parser for Sysmon for Windows
+  Version: '0.5.1'
+  LastUpdated: June 7 2024
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmonWindowsEvent.yaml:4]: Done

File: Parsers/ASimDns/Parsers/ASimDns.yaml [hunk 11]
```diff
@@ -27,6 +27,7 @@ Parsers:
   - _ASim_Dns_MicrosoftNXlog
   - _ASim_Dns_MicrosoftOMS
   - _ASim_Dns_MicrosoftSysmon
+  - _ASim_Dns_MicrosoftSysmonWindowsevent
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/ASimDns.yaml:30]: Done

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml [hunk 12]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml:13]: Done

File: Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml [hunk 13]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: DNS activity ASIM filtering parser for Sysmon for Windows
-  Version: '0.4'
-  LastUpdated: Jan 3 2023
+  Version: '0.5.1'
+  LastUpdated: June 7 2024
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml:4]: Done

File: Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmon.yaml [hunk 14]
```diff
@@ -13,7 +13,7 @@ References:
 - Title: ASIM
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmon.yaml:13]: Done

File: Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmon.yaml [hunk 15]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: DNS activity ASIM parser for Sysmon for Windows
-  Version: '0.4'
-  LastUpdated: Jan 3 2023
+  Version: '0.5.1'
+  LastUpdated: June 7 2024
```
[the remark this answers is not in the document]
Comment (Alekhya0824) [reply Parsers/ASimDns/Parsers/ASimDnsMicrosoftSysmon.yaml:4]: Done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
One suggestion: once you're done with the open comment change, please click on the `Resolve Conversation` button. Thanks

**Review by vakohl — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**